File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/Link.java
package com.raxim.myscoutee.profile.data.dto.rest;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Link {
    @JsonProperty(value = "link")
    private com.raxim.myscoutee.profile.data.document.mongo.Link link;

    @JsonProperty(value = "info")
    private LinkInfo info;

    public com.raxim.myscoutee.profile.data.document.mongo.Link getLink() {
        return link;
    }

    public void setLink(com.raxim.myscoutee.profile.data.document.mongo.Link link) {
        this.link = link;
    }

    public LinkInfo getInfo() {
        return info;
    }

    public void setInfo(LinkInfo info) {
        this.info = info;
    }
}

File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/Link.kt
package com.raxim.myscoutee.profile.data.dto.rest

import com.fasterxml.jackson.annotation.JsonProperty

data class Link(

    @field:JsonProperty(value = "link")
    val link: com.raxim.myscoutee.profile.data.document.mongo.Link? = null,

    @field:JsonProperty(value = "info")
    val info: LinkInfo? = null
)